sqlmesh/core/context.py

```diff
@@ -2037,9 +2036,10 @@ def _run_plan_tests(
                     self._test_connection_config._engine_adapter.DIALECT,
                 )
             if not result.wasSuccessful():
-                raise PlanError(
-                    "Cannot generate plan due to failing test(s). Fix test(s) and run again"
+                self.console.log_error(
```

Comment:
Wouldn't the `PlanError` be intercepted in the cli's error handler and we end up printing the error twice? Why not just raise and let it print there?